From: Alice Mikhaylenko <alicem@mailbox.org>
Date: Wed, 23 Apr 2025 20:58:15 +0000
Subject: header-bar: Fix the icon padding for .default-decoration too

Missed that one.


(cherry picked from commit 98643bce8edf94d50add0b72fff2024742390507)

Co-authored-by: Alice Mikhaylenko <alicem@gnome.org>

Origin: https://gitlab.gnome.org/GNOME/libadwaita/-/commit/37411631c52c397a0cd
Bug-Ubuntu: https://bugs.launchpad.net/ubuntu/+source/yaru-theme/+bug/2106784
---
 src/stylesheet/widgets/_header-bar.scss | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/src/stylesheet/widgets/_header-bar.scss b/src/stylesheet/widgets/_header-bar.scss
index 7166931..87f944c 100644
--- a/src/stylesheet/widgets/_header-bar.scss
+++ b/src/stylesheet/widgets/_header-bar.scss
@@ -56,6 +56,10 @@ headerbar {
         min-height: 22px;
         min-width: 22px;
         padding: 4px;
+
+        > image {
+          padding: 3px;
+        }
       }
 
       > .icon {
